jmeter-service/testrunner.go

```diff
@@ -149,7 +152,7 @@ func (tr *TestRunner) runHealthCheck(testInfo TestInfo, testStartedAt time.Time,
 		if err := tr.sendTestsFinishedEvent(testInfo, testStartedAt, msg, keptnv2.ResultFailed); err != nil {
 			logger.Errorf(errMsgSendFinishedEvent, err, testInfo)
 		}
-		return nil
+		return errors.New(msg)
```

Comment:
I think we can keep `return nil` here, as a failed jmeter test is not necessarily an error in the execution of the `test` task

Reply:
When leaving nil here, we see that the finished event is sent in this place and then also in sendTestResult() function

Reply:
ok, thx for clarifying :)